### DISCUSSION TITLE
Add a toggle for private sessions and have a default thumbnail for movies/music if its missing

### DISCUSSION THREAD

**Discussion by Sculside** [Radiicall/jellyfin-rpc#143] [Ideas]
I just found this today and got it set up, very great work on it! I don't know how viable these would be to add but I had a couple ideas while using the app. The first thing that would be useful is a toggle to hide the session similar to Spotify's "Private session" setting. Currently the only way I see to hide the status is remove the developer app on Discord or mark yourself as offline but I could be missing something.

The second thing I noticed is that if a song I'm listening to doesn't have cover art, the missing image icon shows up as seen in the attached photo. (I'm using port forwarding instead of imgur) If there was a default icon that could be shown if there's no cover art that would fix the odd looking icon in the user profile.
![Screenshot 2024-08-21 143813](https://github.com/user-attachments/assets/3cf3b951-2872-4521-bef0-57fede37bbfe)

**Comment by justradical** [comment]
The image problem was fixed in a previous version but didnt get ported to the rewrite, ill get that fixed again asap. I'm not entirely sure about the whole private mode thing or how i would implement that, config is only loaded on launch and there is no status icon or application window where you could toggle it

**Reply by Sculside** [reply]
Thank you for the quick reply! Glad it wasn't an error on my part and good to hear it's an easy fix! When I was looking at ways to hide it it makes sense why it would be a challenge to implement a private mode even with my limited coding knowledge but it's an idea if you want to try and make it possible. Good luck if you decide to work on it and thank you for making the program!
